sympy/printing/pretty/tests/test_pretty.py

```diff
@@ -1087,79 +1084,6 @@ def test_pretty_relational():
     assert pretty(expr) in [ascii_str_1, ascii_str_2]
     assert upretty(expr) in [ucode_str_1, ucode_str_2]
 
-def test_Assignment():
```

Comment:
Why were these tests removed? I think the pretty printing should be maintained, even if the class heirarchy changes.

Reply:
The old class inherited from `Relational` and thus was printed like `x := y + 1`. The new class just inherits string printing behavior from the base AST class. I put the new tests for `repr(Assignment(...))` in `codegen/tests/test_ast.py`, which follows the pattern for the module.

Reply:
I don't see why it shouldn't continue to pretty print with := with the new class structure.